### ISSUE TITLE
Decode latent to smaller size than 512x512?

### ISSUE THREAD

**Issue by GLivshits** [huggingface/diffusers#4202]
Hello. I am currently conducting a research on image manipulation with Stable Diffusion, which involves CLIP score calculation.
I do start with latent of size 64x64, pass it through decoder (and get 512x512 image), resize it to 224x224 and then pass to CLIP.
However, there is one drawback: since I am calculating gradients, decoding stage occupies A LOT of memory. 
Since I am doing downsampling anyways, it would be nice to decode latent not into 512x512 image, but into 256x256.
I've tried simple downsampling using torch.nn.functional.interpolate, but such approach produces image with lots of artifacts.

It would be nice to have a way to downsample the latent in order to use some classifier-guidance methods.
If anybody knows such a way - let me know.

**Comment by eliphatfs** [comment]
IMO this is not possible without fine-tuning the VAE.

**Comment by sayakpaul** [comment]
As correctly mentioned by @eliphatfs, it's not possible without fine-tuning the VAE. Closing this issue henceforth.
